include/can/core/messages.hpp

```diff
@@ -465,6 +468,27 @@ struct MotorPositionResponse : BaseMessage<MessageId::motor_position_response> {
     auto operator==(const MotorPositionResponse& other) const -> bool = default;
 };
 
+// This response is the exact same payload as MotorPositionResponse
+struct UpdateMotorPositionResponse
```

Comment:
nit you can ignore. As I understand it, this message is for the instances where you might want to read the current position without having issued a `Move`. Could we instead call this something like `GetCurrentMotorPosition` or something to that effect?

Reply:
The `UpdateMotorPositionRequest` command is used to update the position tracker in the firmware, and this is the response to it. I don't think that `GetCurrentMotorPosition` is an accurate representation of what the command does, and the functionality you're describing exists as `MotorPositionRequest`. 

Disambiguating this command by naming it `UpdateMotorPositionFromEncoder` would make sense to me!